### ISSUE TITLE
Please add DMS conversion and inverse (Degree minute seconds)

### ISSUE THREAD

**Issue by jgordini** [abrudz/aplcart#99]
Edit: From discussions on APL orchard it looks like allowing Negative Degree, Negative Minute, Negative Second is the best route. 

[Link to post on APL Orchard](https://chat.stackexchange.com/transcript/message/61865388#61865388)

---------- Previous Note ------
The [Wikipedia Degree page](https://en.wikipedia.org/wiki/Decimal_degrees#Example) also references [ISO standard 6709](https://en.wikipedia.org/wiki/ISO_6709#Latitude) and in these examples only the degree portion can be positive or negative.

However I am unsure whether negating the minutes and seconds would create problems when doing additional calculations. 

This also may only be specific to problems of latitude and longitude. 

-----Previous Note End -----

**Suggested additional code entry** (required)
```
0 60 60⊤3600×42.125

Either
(××0 60 60⊤3600×|)N
(3600÷⍨60∘⊥)N
or
(×(×@1)0 60 60⊤3600×|)N
(3600÷⍨×∘⊃×60⊥|)N
```

**Description** (required)
```
Conversion from Decimal Degrees to DMS (Degrees Minutes and Seconds)
```

**Class** (optional)
```
Tacit | Dfn | Dop | Primitive | Syntax | Non-syntax | System | I-beam | Constant | …
```

**Type** (optional)
```
Dyadic Function | Monadic Function | Function | Array | Assignment | Command | Ucmd | …
```

**Group** (optional)
```
Dyadic Operator | Monadic Operator | Structure | Keyword | UCMD GROUP | …
```

**Category** (required)
```
Array Properties | Boolean/Logical | Comparison | Data Conversion | Expression | Function Application | Index Generation | Mathematical | Selection | Sets | Structural | System | Text
```

**Keywords** (optional)
```
Mixed Radix, Degrees, DMS
```
**TIO (optional)**
```
Try it online URL https://tio.run/##SyzI0U2pTMzJT////1Hf1EdtEzQOTz883UDBDIQedS0xNjMwODy9RtPESM/QyPT/fwA
```
---
**Note:** By submitting this issue, you permit your contribution to be [used without restriction](../../blob/master/LICENSE).

**Comment by abrudz** [comment]
Fixed in 75ecd605cec3f8b5e9edd3dbd171359834d3f424

**Comment by jgordini** [comment]
Thanks!
